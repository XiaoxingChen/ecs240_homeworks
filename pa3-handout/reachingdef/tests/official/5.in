c Branches
p 10 10 20 1 10
b 1 
e 1 2
e 1 3
e 1 4
e 1 7
b 2 
e 2 3
e 2 4
e 2 6
e 2 7
e 2 8
b 3 
e 3 4
e 3 5
b 4 0 8
e 4 9
e 4 5
e 4 7
b 5 2 1 4 6 3 1 5 8 3 8
e 5 9
e 5 7
b 6 5 10 4 3 3 7
e 6 9
b 7 
e 7 10
b 8 
e 8 9
b 9 
e 9 10
b 10 
