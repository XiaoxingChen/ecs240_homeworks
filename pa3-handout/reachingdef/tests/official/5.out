rdout 1
rdout 2
rdout 3
rdout 4
rdout 5 5
rdout 6 6
rdout 7 5
rdout 8
rdout 9 5 6
rdout 10 5 6
